spark/v4.0/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestSystemFunctionPushDownInRowLevelOperations.java

```diff
@@ -260,7 +267,12 @@ private void checkUpdate(RowLevelOperationMode mode, String cond) {
               DistributionMode.NONE.modeName());
 
           Dataset<Row> changeDF = spark.table(tableName).where(cond).limit(2).select("id");
-          changeDF.coalesce(1).writeTo(tableName(CHANGES_TABLE_NAME)).create();
+          try {
+            changeDF.coalesce(1).writeTo(tableName(CHANGES_TABLE_NAME)).create();
+          } catch (TableAlreadyExistsException e) {
+            throw new AlreadyExistsException(
```

Comment:
```
  @throws(classOf[TableAlreadyExistsException])
  def create(): Unit
```
This API now throws `TableAlreadyExistsException`

Reply:
Why do we need to change the exception type?

Reply:
if I re-throw without doing anything
```
          try {
            changeDF.coalesce(1).writeTo(tableName(CHANGES_TABLE_NAME)).create();
          } catch (TableAlreadyExistsException e) {
            throw e;
          }
```
I got `Unhandled exception: org.apache.spark.sql.catalyst.analysis.TableAlreadyExistsException`

Reply:
Ah got it!